credentials.go

```diff
@@ -0,0 +1,243 @@
+package main
+
+import (
+	"context"
+	"encoding/base64"
+	"errors"
+	"fmt"
+	"log"
+	"net/url"
+	"strings"
+
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/ecr"
+	"github.com/aws/aws-sdk-go-v2/service/s3"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	endpoints "github.com/aws/smithy-go/endpoints"
+	"github.com/aws/smithy-go/middleware"
+	smithyhttp "github.com/aws/smithy-go/transport/http"
+	"github.com/google/go-containerregistry/pkg/authn"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/tools/clientcmd/api"
+)
+
+type Oracle struct {
+	RegistryAddr string
+	RegistryUser string
+	RegistryPass string
+	ClusterIP    string
+	Token        string
+	CaCert       string
+}
```

Comment:
![image](https://github.com/zetane/ZetaForge/assets/35940434/5ec630f6-f326-4bed-a467-4c0bf916e52a)

Reply:
It was already there I swear.